src/views/rss/show.slang

```diff
@@ -8,6 +8,10 @@ rss version=2.0
       item
         title = announcement.title
         description = announcement.content
+          | <small>#{announcement.typename}</small>
```

Comment:
Add a dot at the end, so it will separate the sentence.

Reply:
What's do a dot at the end, I can't find it in [doc](https://github.com/jeromegn/slang).

Something like this:
```
| <small>#{announcement.typename}</small>
```

Reply:
Sorry, i meant that the following sentence confuses a bit:

```
Blog post Announced a week ago by veelenga
```

Would be really better to have this instead:

```
Blog post. Announced a week ago by veelenga
```

Reply:
@veelenga Sorry, I already delete my comment because I understated what you mean. Thank you for your answer :)